spec/ember-cli/manifest_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "ember-cli/manifest"
+
+describe EmberCli::Manifest do
+  describe "#fixup!" do
+    it "fixes paths" do
+      input = build_manifest("ember", {
+        "assets" => {
+          "frontend.js" => "frontend-abc.js",
+        },
+        "files" => {
+          "frontend-abc.js" => {
+            "logical_path" => "frontend.js",
+          }
+        }
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.<br>Put a comma after the last parameter of a multiline method call.